### DISCUSSION TITLE
Threads.netがnodeinfoに対応していないので概要がひどいことになっている

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [misskey-dev/misskey#14043] [General]
Threads.net のサーバー概要は、現状次のとおり表示されます:

![image](https://github.com/misskey-dev/misskey/assets/48310258/5d20b56d-960b-4c41-b75d-44edb9cd5dbc)

ソフトウェアのバージョンは仕方ないとして、一貫性のためにそれ以外のプロパティへ何らかの意味のある値を注意書き付きで入れてほしいと思います。

**Comment by anatawa12** [comment]
不明が正しいけど(不明)((不明))は(不明)一つにしたほうがよさそう
